packages/test-utils/src/wrapper-array.js

```diff
@@ -5,12 +5,21 @@ import type VueWrapper from './vue-wrapper'
 import { throwError, warn } from 'shared/util'
 
 export default class WrapperArray implements BaseWrapper {
-  wrappers: Array<Wrapper | VueWrapper>;
-  length: number;
+  +wrappers: Array<Wrapper | VueWrapper>;
+  +length: number;
 
   constructor (wrappers: Array<Wrapper | VueWrapper>) {
-    this.wrappers = wrappers || []
-    this.length = this.wrappers.length
+    const length = wrappers.length
+    // $FlowIgnore
+    Object.defineProperty(this, 'wrappers', {
+      get: () => wrappers,
+      set: () => {}
```

Comment:
We should throw an error to tell the users it's read only.

I realize this isn't part of this PR, but I think we should do the same for vnode, and element.

Reply:
`vnode` is undocmumented.
Should `vnode` is added to document?

Reply:
Thanks. No I don't think it needs to be documented

Reply:
Thank you for reply.
I think so too.